### PR TITLE
Migrate Meraki modules to collections

### PR DESCRIPTION
**Pull request by kbreit**
##### SUMMARY
This pull request tells BOTMETA the Meraki modules have been migrated to Galaxy. Please let me know if there are additional fields I need to fill out.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
meraki

### CODE REVIEW

**Comment by gundalow** [comment]
@kbreit Hi, thanks for raise this and migrating to collections.

Could you please expand `BOTMETA` so that it lists every module and plugin with `migrated_to:`, this is so docs.ansible.com/ansible can link from the old module docs to the new location. Once the modules & plugins are `git rm`'ed from ansible/ansible without this we will no longer have a list of the files to link.

**Comment by gundalow** [comment]
See https://github.com/ansible/ansible/pull/65885 for an example

Could you please move the change to `docs/docsite/rst/dev_guide/developing_collections.rst` into a separate PR, thanks :)

**Comment by kbreit** [comment]
@gundalow I applied the changes you requested. Please let me know if there's anything more I need to do. Thank you.

**Comment by kbreit** [comment]
@gundalow I am seeing the error on line 338 but not sure what this is for. Is this the `&meraki` phrase?

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -335,9 +335,50 @@ files:
   $modules/network/icx/: sushma-alethea
   $modules/network/layer2/: $team_networking
   $modules/network/layer3/: $team_networking
+  $modules/network/meraki/meraki_admin.py: &meraki
```
Comment (gundalow) [inline .github/BOTMETA.yml]: The `*meraki` means use the YAML anchor, which you defined on line 378


So you only need to list the files like this:
```
  $modules/network/meraki/meraki_admin.py: *meraki
  $modules/network/meraki/meraki_config_template.py: *meraki
```

Comment (kbreit) [reply .github/BOTMETA.yml]: I made some changes which pulled some of the anchor references. Not sure if this is right, but please take a look. Thanks.

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
Thanks Kevin!

**Pull request merged**